torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -525,13 +642,13 @@ struct TensorExprKernel {
               }));
       buffer_args.push_back(std::move(in_buffer));
     }
-
+    bool skip_llvm_codegen = false;
```

Comment:
Let's not add this logic for skipping llvm codegen, it makes the control flow below way more complicated.  We either want to build with llvm and use it, or not.

Reply:
resolved